book/src/developers/contributing/releases/deployment.md

```diff
@@ -1,37 +1,100 @@
-# Deployment
+# Deploy trin to network
 
-## Update testnet node images
-Run `make create-docker-image` and `make push-docker-image` commands with the appropriate version.
-
-Example:
-
-```sh
-make create-docker-image version=0.2.0-alpha.3
-make push-docker-image version=0.2.0-alpha.3
+## First time Setup
+- Get access to cluster repo (add person to @trin-deployments)
+- Download cluster repo:
+```shell=
+git clone git@github.com:ethereum/cluster.git
+cd cluster
+python3 -m venv venv
+. venv/bin/activate
+pip install ansible
 ```
+- Publish your pgp public key with keybase, using: `keybase pgp select --import`
+  - This fails if you don't have a pgp key yet. If so, create one with `gpg --generate-key`
+- [Install sops](https://github.com/ethereum/cluster/tree/master/portal-network/trin/ansible)
+- Contact `@paulj`, get public gpg key into cluster repo
+- Make sure your pgp key is working by running:
+  ```sops portal-network/trin/ansible/inventories/dev/group_vars/secrets.sops.yml```
+- Log in to docker with: `docker login`
+- Ask Nick to be added as collaborator on Docker repo
 
-## Deploy testnet nodes
+## Each Deployment
 
-Run the Ansible playbook to fetch the newly available docker image and update the testnet nodes.
+### Prepare
+- Announce in Discord #trin that you're about to run the deployment
+- Make sure to schedule plenty of time to react to deployment issues
 
-## Communicate
+### Update docker images
+Docker images are how Ansible moves the binaries to the nodes. Update the docker tags with:
```

Comment:
It might be worth leaving a short note here about what tags are used for what. eg. 

> Our CI pushes the `latest` & `latest-bridge` images to the docker registry for every pr that's merged into master. Our network uses the `testnet` & `bridge` tags to build from. So, to deploy, we need to pull the latest `latest` & `latest-bridge` tags, rename them, and then push the newly tagged images to the registry.

Ok, that might be a little wordy. But, I'm just thinking it might be helpful to explain why we do these steps so people can have a reference in case something unexpected happens.